src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/DraftReferralDTOTest.kt

```diff
@@ -43,16 +43,13 @@ class DraftReferralDTOTest(@Autowired private val json: JacksonTester<DraftRefer
     val referral = Referral(
       id = UUID.fromString("3B9ED289-8412-41A9-8291-45E33E60276C"),
       createdAt = OffsetDateTime.parse("2020-12-04T10:42:43+00:00"),
-      createdByUserID = "123456",
-      createdByUserAuthSource = "delius",
     )
     val out = json.write(DraftReferralDTO.from(referral))
     assertThat(out).isEqualToJson(
       """
       {
         "id": "3b9ed289-8412-41a9-8291-45e33e60276c", 
-        "createdAt": "2020-12-04T10:42:43Z",
-        "createdByUserId": "123456"
+        "createdAt": "2020-12-04T10:42:43Z"
```

Comment:
_not a blocker_
Is it worth using microseconds (as used by PostgreSQL) precision here?